pkgs/tools/security/gnupg/20.nix

```diff
@@ -20,6 +20,7 @@ stdenv.mkDerivation rec {
 
   patchPhase = ''
     find tests -type f | xargs sed -e 's@/bin/pwd@${coreutils}&@g' -i
+    find . -name pcsc-wrapper.c | xargs sed -i 's/typedef unsinged int pcsc_dword_t/typedef unsigned int pcsc_dword_t/'
```

Comment:
Please apply that patch only on Darwin, i.e. add `++ optionalString stdenv.isDarwin ''...''` after the original `patchPhase` value. This avoids side-effects and re-builds on other platforms.